astropy/units/tests/test_quantity_non_ufuncs.py

```diff
@@ -2578,3 +2585,170 @@ def test_all_included(self):
     @needs_array_function
     def test_ignored_are_untested(self):
         assert IGNORED_FUNCTIONS | TBD_FUNCTIONS == untested_functions
+
+
+@pytest.mark.parametrize(
+    "functions",
```

Comment:
I'm fairly sure it would work to write `"target, helper"` here, so that the signature of all the test functions can be `test_...(self, target, helper)`, which I think would be clearer.